.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
```

Comment:
import/extensions, import/no-unresolved, global-require seems to be useless for the project, they should be turned off.
Like this: 

"rules": {
        "import/extensions": "off",
        "import/no-unresolved": "off",
        "global-require": "off"
    },

Reply:
Agreed on first two, global-require is not really an allowed rule globally.